x-pack/test/security_solution_cypress/config.ts

```diff
@@ -46,6 +46,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
         '--xpack.ruleRegistry.unsafe.legacyMultiTenancy.enabled=true',
         `--xpack.securitySolution.enableExperimental=${JSON.stringify([
           'chartEmbeddablesEnabled',
+          'alertSuppressionForIndicatorMatchRuleEnabled',
```

Comment:
I saw you are already passing this value to the test itself, so please, remove this configuration from here before merging :)

Reply:
after I removed it, locally running tests start to fail.
`cypress:open:ess` and `cypress:open:serverless`
Since feature flag was not applied. If I remember correctly, you mentioned, config in file would only work for qa host or on CI. And we still need it locally.

Reply:
We synced about this issue over a call:

* @vitaliidm wants to have the configuration in both places because Cypress open does not read from the config file and he wants for the test to work right away without having to manually add it when executing the test in local.
* I wanted to have it just in once place to avoid future missunderstandings when adding new tests. I.E. someone thinking that the configuration is not added globally because is added just on the test.

After understanding @vitaliidm's point of view I agree and we talked about adding a comment to the spec file to make sure we give visibility to the fact that the configuration is used globally.